internal/statestore/backfill.go

```diff
@@ -208,6 +217,17 @@ func (rb *redisBackend) UpdateBackfill(ctx context.Context, backfill *pb.Backfil
 	return nil
 }
 
+func isBackfillExpired(conn redis.Conn, id string, ttl time.Duration) (bool, error) {
```

Comment:
May be it is better to remove last argument as it is not changed anywhere and the ttl duration is always has one value.

Reply:
Then I need to link this function with redisBackend struct.
I will leave it as is.